src/main/java/io/aiven/connect/jdbc/source/TimestampIncrementingTableQuerier.java

```diff
@@ -83,12 +85,17 @@ public TimestampIncrementingTableQuerier(final DatabaseDialect dialect,
                                              final String incrementingColumnName,
                                              final Map<String, Object> offsetMap,
                                              final Long timestampDelay,
+                                             final List<Long> initialsOffsets,
                                              final TimeZone timeZone) {
         super(dialect, mode, name, topicPrefix);
         this.incrementingColumnName = incrementingColumnName;
         this.timestampColumnNames = timestampColumnNames != null
             ? timestampColumnNames : Collections.<String>emptyList();
         this.timestampDelay = timestampDelay;
+        final long initialTimestampOffset = initialsOffsets.get(0);
+        final long initialIncrementingOffset = initialsOffsets.get(1);
```

Comment:
Because Checkstyle has max param count 10 check, https://github.com/aiven/aiven-kafka-connect-jdbc/blob/master/checkstyle/checkstyle.xml#L280. I know, it doesn't look nice to have that list. But the options are 1) relax the CS test, 2) do this what there is now, 3) refactor the code and e.g. add some POJO wrapper(s) for the params.

Reply:
Yeah, right, Checkstyle
We can do this:
```
diff --git a/checkstyle/suppressions.xml b/checkstyle/suppressions.xml
index 61eff17..733bec6 100644
--- a/checkstyle/suppressions.xml
+++ b/checkstyle/suppressions.xml
@@ -37,7 +37,7 @@
               files="(DataConverter|GenericDatabaseDialect|JdbcSourceTask).java"/>
 
     <suppress checks="ParameterNumber"
-              files="(ColumnDefinition|GenericDatabaseDialect|SqlServerDatabaseDialect).java"/>
+              files="(ColumnDefinition|GenericDatabaseDialect|SqlServerDatabaseDialect|TimestampIncrementingTableQuerier).java"/>
     <!---->
 
 
```

There are some exceptions already, so refactoring will be needed at some point...

Reply:
I see, didn't notice that there's such a fine-grained suppression mechanism. I'll use that and two separate params.

Reply:
Added a fine-grained exception

```
    <suppress checks="ParameterNumber"
              files="TimestampIncrementingTableQuerier.java"
              lines="80"/>
```